### ISSUE TITLE
S3 create bucket policy not returning XML

### ISSUE THREAD

**Issue by vangorra** [localstack/localstack#382]
I'm using node to set the policy document of an S3 bucket hosted in localstack. I noticed when setting the policy document, neither XML nor JSON is returned. Only "True".

Running this in node:
``` javascript
const r = new AWS.S3().putBucketPolicy({
  Bucket: 'bucketName',
  Policy: JSON.stringify({
    Version: '2012-10-17',
    Statement: [
      Action: [ 's3:GetObject' ],
      Effect: 'Allow',
      Resource: 'arn:aws:s3:::bucketName/*',
      Principal: { AWS: [ '*' ] }
    ]
  }),
});

// This is where we see the raw HTTP Response. It's "True".
r.on('httpDone', (response: any) => {
  console.log('HTTP Response', response.httpResponse.body.toString());
});

await r.promise();
```

Produces this error:
```
XMLParserError: Non-whitespace before first tag.
Line: 0
Column: 1
Char: T
```
Notice how the string "True" is trying to be parsed as XML. The aws node api assume the response will be XML. So it should probably be provided.

I would expect the service to return valid XML. That would be consistent with the rest of the AWS API.

**Comment by whummer** [comment]
Thanks for reporting @vangorra , this should be fixed in #384 . Can you please give it a try with the latest version? Thanks

**Comment by vangorra** [comment]
Confirmed fixed. Thanks.
